Indicators/ChandeKrollStop.cs

```diff
@@ -0,0 +1,131 @@
+/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using QuantConnect.Data.Market;
+using System;
+using System.Linq;
+
+namespace QuantConnect.Indicators
+{
+    /// <summary>
+    /// This indicator computes the short stop and lower stop values of the Chande Kroll Stop Indicator.
+    /// It is used to determine the optimal placement of a stop-loss order.
+    /// </summary>
+    public class ChandeKrollStop : BarIndicator, IIndicatorWarmUpPeriodProvider
+    {
+        private readonly AverageTrueRange _atr;
+        private readonly decimal _atrMult;
+        private readonly RollingWindow<IBaseDataBar> _inputValues;
+        private readonly RollingWindow<decimal> _high_stop_list;
+        private readonly RollingWindow<decimal> _low_stop_list;
+
+        /// <summary>
+        /// Gets the short stop of ChandeKrollStop.
+        /// </summary>
+        public IndicatorBase<IndicatorDataPoint> ShortStop { get; }
+
+        /// <summary>
+        /// Gets the long stop of ChandeKrollStop.
+        /// </summary>
+        public IndicatorBase<IndicatorDataPoint> LongStop { get; }
```

Comment:
Maybe we can just rename LongStop->Long & ShortStop->Short

Reply:
ca1720 / identifier contains type name

so keeping as it is